recipes/cotila/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class CotilaConan(ConanFile):
+    name = "cotila"
+    description = "A compile time linear algebra system"
+    homepage = "https://github.com/calebzulawski/cotila"
+    topics = ("c++", "math", "linear algebra", "compile-time")
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _min_cppstd(self):
```

Comment:
This is not used.
Did you mean to add a `validate` method checking it using `tools.check_min_cppstd`?